app/src/main/kotlin/com/wire/android/ui/home/newconversation/common/SelectParticipantsButtonsRow.kt

```diff
@@ -178,23 +177,30 @@ fun SelfDeletionTimerButton(
             .padding(start = dimensions().spacing16x)
             .size(dimensions().spacing48x)
             .clip(RoundedCornerShape(size = dimensions().onMoreOptionsButtonCornerRadius))
-            .background(
-                color = if (isSelected) colorsScheme().onSecondaryButtonSelected else colorsScheme().secondaryButtonEnabled,
-            )
     ) {
         WireSecondaryButton(
             leadingIcon = {
                 Image(
                     painter = painterResource(id = R.drawable.ic_timer),
                     contentDescription = stringResource(id = R.string.content_description_self_deletion_selector_button),
                     colorFilter = ColorFilter.tint(
-                        if (isSelected) colorsScheme().secondaryButtonSelected else colorsScheme().onSecondaryButtonEnabled
-                    ),
+                        when {
```

Comment:
Maybe I'm wrong but I think icons should be tinted automatically, we shouldn't modify the `colorFilter` as the theme should be responsible for applying proper color to the `leadingIcon`.

Reply:
I tried not applying any colorFilter and leaving the responsibility as you said to the internal colorButtons parameters, but the icon color was never changed no matter the state 🤷